### ISSUE TITLE
Updates

### ISSUE THREAD

**Issue by R4Z0R49** [R4Z0R49/DayZStats#3]
Looks like this needs some updating :-) maybe made a bit more simple.

http://198.27.83.122/SurvivorStats/index.php?Main

:-)

**Comment by Marcuzz** [comment]
I agree, I don't think it was ever fully finished either. And that page looks a lot better than this currently does. Does anyone even use this still?

**Comment by leegreaves** [comment]
Hey Marcuzz...guess what...someone does still use it. I run a small private hive epoch server on a dedi box and been researching some stats I could use as part of a webpage.Ive narrowed it down to three, but each of them have their pro's and their con's. With the link above I do notice stats are working for it correctly, but for me, not much luck. Ive had to change some of the code as the epoch tables show labels in caps ie Player_DATA. On my database however they were showing all lowercase ie player_data. Ive changed everything regarding this to my full knowledge but seems that it isnt pulling data from the db correctly. Im wondering if you contact n8m4rede to see if you can take over this repo and see what magic you and a few other people could come up with.

The other web stats pages I were researching was the Epoch Server Stats by mudzerelli and also R4Z0R49's own stats page - BUT this one in particular will prove more of a barrior to overcome as from what I can make out it was designed for the original dayz mod and not for epoch, so hence trying to port it for epoch is more of a problem as there are numerous errors being reported.

If you wanna see what I mean then of course PM me and I can provide links to these pages and perhaps with a little persuasion you could either take on the project and see what can be made from it. Perhaps eventually using it as a basis for dayz standalone (eventually) when servers are allowed to link stat pages to public/private servers.
